### DISCUSSION TITLE
Translation Contributions

### DISCUSSION THREAD

**Discussion by IraSoro** [IraSoro/peri#284] [Q&A]
### Hello, contributors! 

We are looking for your help to translate our app into different languages! Your contributions will make our project more accessible to users around the world.

#### Instructions for Contributions
You can find detailed instructions on how to submit a translation using a Pull Request in the [CONTRIBUTING.md](https://github.com/IraSoro/peri/blob/master/CONTRIBUTING.md) file. However, if you prefer not to create a Pull Request, you can simply leave your translation in the comments below.

#### How to Contribute:
- **Attach your translation file**: If you have a translation ready, please attach the file here. Make sure it follows the format of our existing translation files. (See example below)
- **Leave a comment with your translation**: If you prefer, you can also write your translation directly in the comments below.

#### Example:
You can create a new file for the language by duplicating the [`./src/utils/translations/example.ts`](https://github.com/IraSoro/peri/blob/master/src/utils/translations/example.ts).  All the phrases for translation are written there with examples, so you can easily learn how to use them in context.

For reference, here’s an example of a translated file into [Russian](https://github.com/IraSoro/peri/blob/master/src/utils/translations/ru.ts) and [Spanish](https://github.com/IraSoro/peri/blob/master/src/utils/translations/es.ts).

#### Existing Translations:
We currently have translations stored in the [`translations`](https://github.com/IraSoro/peri/tree/master/src/utils/translations) folder. Feel free to check them out for consistency.

#### Important Note:
Don’t hesitate to share your ideas, even if they’re still in progress. We appreciate every contribution!

**Comment by IntegerAlex** [comment]
I want to Translate this in Hinglish ( which is Hindi+English) and in India majority of women/people understand Hinglish well.
 I will cross verify with my team and create a pull request, any more details you need let me know

**Reply by IraSoro** [reply]
Hello! Thanks for you contribution! We have received your PR to add the language and currently working on it.

**Comment by Darpana-iitgn** [comment]
Hi @IraSoro I have translated this to gujarati, but because of some issues with my computer i am unable to build the project and send you a pull request. Can you tell me how can i share my contribution and get it counted for hacktoberfest?

**Reply by imblowfish** [reply]
Hello, @Darpana-iitgn
Your contribution to the project will be counted by hacktoberfest if PR with your changes will be merged into the project and task will be labeled with hacktoberfest label

If you're unable to push your changes according to PR tutorial from github then we can do it like this:
- Create fork
- Clone your fork locally with `git clone`
- Add your translation locally
- Create Pull request
- Use `git format-patch` command to generate text file with all your changes
- Insert your file in the PR comments

On our side we'll apply your patch file and push this changes in your branch

But it's much better if you'll try to fix your problems. Maybe I can help with your local building and Pr creation problems?

**Comment by godlykmathews** [comment]
Hi @IraSoro I can translate to Malayalam

**Reply by IraSoro** [reply]
Hello @godlykmathews! Thank you for your interest in our project! If you want to contribute you can just make a PR, and I will link it to the issue and set the necessary Hacktoberfest tags.
